src/cli/remotes-wizard.js

```diff
@@ -2,22 +2,28 @@ import qrcode from 'qrcode-terminal';
 import chalk from 'chalk';
 import log from './log';
 import promisifyEvent from 'promisify-event';
+import dedent from 'dedent';
 
 export default async function (testCafe, remoteCount, showQRCode) {
     var connections = [];
 
     if (remoteCount) {
         log.hideSpinner();
+        log.write(dedent`
```

Comment:
Use `dedent` as function instead of tag (babel has a nasty bug in the latest version, which breaks template tags)